lib/src/utils/crypto/extension/uint8list_extension.dart

```diff
@@ -0,0 +1,9 @@
+
+import 'dart:convert';
+import 'dart:typed_data';
+
+extension Unit8ListExtension on Uint8List {
+  String toBase64() {
+    return base64Url.encode(this).replaceAll('=', '');
```

Comment:
why we need remove `=` char

Reply:
I thought that the '=' character is reserved for special meaning for file system name or URLs, but i don't use base64, base64url in any of that. So, year, can be removed

Reply:
Would that corrupt the base64 data?

Reply:
because matrix using unpadded base64 https://spec.matrix.org/v1.6/appendices/#unpadded-base64

Reply:
> Would that corrupt the base64 data?

no. woundn't